src/plugins/bfetch/common/util/normalize_error.ts

```diff
@@ -7,13 +7,19 @@
  */
 
 import { ErrorLike } from '../batch';
+import { BfetchRequestError } from '..';
 
 export const normalizeError = <E extends ErrorLike = ErrorLike>(err: any): E => {
   if (!err) {
     return {
       message: 'Unknown error.',
     } as E;
   }
+  if (err instanceof BfetchRequestError) {
+    // ignoring so we can return the error as is
+    // @ts-expect-error
```

Comment:
I really dislike this code which makes me more comfortable with this `ts-expect-error` - Errors are being transformed in such a way as they lose their original class type AND any attributes aside from 'message'. It would be better to make the inputs conform than to transform all inputs.